### ISSUE TITLE
[no-invalid-aria-attributes] Doesn't work at all?

### ISSUE THREAD

**Issue by boris-petrov** [ember-template-lint/ember-template-lint#2385]
I don't understand - even using a simple `<button type="button" aria-haspopup="true">` leads to `The value for aria-haspopup must be a single token from the following: false, true, menu, listbox, tree, grid, dialog.`. Am I doing something wrong?

**Comment by locks** [comment]
I am also running into a snag with this rule. The following code:

```hbs
<NavTabs aria-label="{{@crate.name}} crate subpages" local-class="nav" as |nav|>
</NavTabs>
```

Triggers this error:

```bash
app/components/crate-header.hbs
  35:0  error  The value for aria-label must be a string.  no-invalid-aria-attributes
```

**Comment by bmish** [comment]
CC: @judithhinlung

**Comment by kpfefferle** [comment]
I am getting the same error with a button that has `aria-haspopup="true"`:

```
error  The value for aria-haspopup must be a single token from the following: false, true, menu, listbox, tree, grid, dialog.  no-invalid-aria-attributes
```

**Comment by judithhinlung** [comment]
The failures for <button aria-haspopup="true"></button> is a bug in the code, which interpreted them as boolean values, instead of valid string values. Casting true/false values to strings before checking that they are part of the list of accepted values will fix this issue.

**Comment by bmish** [comment]
I'd like to publish a fix for this in the next few days if someone can fix it.
